Hydrogent/interface/Tasks/HnPostProcessTask.hpp

```diff
@@ -120,8 +120,6 @@ class HnPostProcessTask final : public HnTask
 
     virtual void Execute(pxr::HdTaskContext* TaskCtx) override final;
 
-    void ResetTAA() { m_ResetTAA = true; }
```

Comment:
Let's not remove this functionality as it still may be useful in certain scenarios.

Reply:
Why do we need it? Storing the previous velocity map, allows us to automatically reset invalid pixels

Reply:
There are some scenarios in which reset may be needed - it does not hurt to have this option.